networkx/utils/tests/test_decorators.py

```diff
@@ -485,7 +481,27 @@ def test_indent(self):
  pass#"""
         )
 
-    def nop(self):
-        print(foo.__argmap__.assemble(foo.__wrapped__))
-        argmap._lazy_compile(foo)
-        print(foo._code)
+    def test_immediate_raise(self):
+        @not_implemented_for("directed")
+        def yield_nodes(G):
+            yield from G
+
+        G = nx.Graph([(1, 2)])
+        D = nx.DiGraph()
+
+        list(yield_nodes(G))
+        print(yield_nodes._code)
```

Comment:
Maybe remove print statement and change first call to `yield_nodes` into a test?  
But the next comment suggests that it is testing the "first call". so maybe the `list(yield_nodes(G))` line should be removed?  Not sure...
```suggestion
        assert list(yield_nodes(G)) == [1, 2]
```

Reply:
*sigh* missed that delta... gone again